packages/components/src/Link/Link.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { PropsWithChildren } from "react";
+
+interface LinkProps {
+  readonly ariaLabel?: string;
+  readonly external?: boolean;
+  readonly url: string;
+}
+
+export function Link({
+  ariaLabel,
+  external = false,
+  url,
+  children,
+}: PropsWithChildren<LinkProps>) {
+  const linkProps = {
+    ...(external && { target: "_blank" }),
+    href: url,
+    ...(ariaLabel && { "aria-label": ariaLabel }),
+  };
+
+  return <a {...linkProps}>{children}</a>;
```

Comment:
Since the props are only used in one place, I suggest this as a bit more direct and easier to read:
```suggestion
  return (
    <a
      href={url}
      {...(external && { target: "_blank" })}
      {...(ariaLabel && { "aria-label": ariaLabel })}
    >
      {children}
    </a>
  );
```